### ISSUE TITLE
Unreliable Build Test: "Cocoapods_Download_Version_Succeeds"

### ISSUE THREAD

**Issue by daalcant** [microsoft/OSSGadget#126]
The build test "Cocoapods_Download_Version_Succeeds" is unreliable. During pipeline runs, the test randomly fails. Re-running the test sometimes passes, sometimes doesn't.

Error:
```
  X Cocoapods_Download_Version_Succeeds [855ms]
  X Cocoapods_Download_Version_Succeeds (pkg:cocoapods/RandomKit,RandomKit.podspec,1) [855ms]
  Error Message:
   No files were downloaded.
  Stack Trace:
     at Microsoft.CST.OpenSource.Tests.DownloadTests.TestDownload(String purl, String targetFilename, Int32 expectedCount) in D:\a\1\s\src\oss-tests\DownloadTests.cs:line 223
   at Microsoft.CST.OpenSource.Tests.DownloadTests.Cocoapods_Download_Version_Succeeds(String purl, String targetFilename, Int32 expectedCount) in D:\a\1\s\src\oss-tests\DownloadTests.cs:line 28
   at Microsoft.VisualStudio.TestPlatform.MSTestAdapter.PlatformServices.ThreadOperations.ExecuteWithAbortSafety(Action action)
```

**Comment by daalcant** [comment]
@scovetta this build test isn't passing at all now, even with local builds. There might be a larger issue here, as there is another download test failing as well:
```
VSM_Download_Version_Succeeds [1m 18s]
  X VSM_Download_Version_Succeeds (pkg:vsm/ms-vscode/PowerShell,extension.vsixmanifest,1) [1m 18s]
  Error Message:
   Directory count 2 does not match expected 1
  Stack Trace:
     at Microsoft.CST.OpenSource.Tests.DownloadTests.TestDownload(String purl, String targetFilename, Int32 expectedCount) in C:\Users\daalcant\repos\OSSGadget\src\oss-tests\DownloadTests.cs:line 230
   at Microsoft.CST.OpenSource.Tests.DownloadTests.VSM_Download_Version_Succeeds(String purl, String targetFilename, Int32 expectedCount) in C:\Users\daalcant\repos\OSSGadget\src\oss-tests\DownloadTests.cs:line 143
   at Microsoft.VisualStudio.TestPlatform.MSTestAdapter.PlatformServices.ThreadOperations.ExecuteWithAbortSafety(Action action)
```

**Comment by gfs** [comment]
@scovetta was this resolved?

**Comment by scovetta** [comment]
Yes, it was resolved via 167da81 in PR #133
